src/clarity-angular/progress/progress-bars/_variables.progress-bars.scss

```diff
@@ -0,0 +1,12 @@
+/*!
+ * Copyright (c) 2016-2017 VMware, Inc. All Rights Reserved.
+ * This software is released under MIT license.
+ * The full license information can be found in LICENSE in the root directory of this project.
+ */
+
+// Usage: ../progress/progress-bars/_progress-bars.clarity.scss
+$clr-progress-defaultBarColor: $clr-action-blue;
```

Comment:
`!default` keyword is from these default variables?

Reply:
Most of the variables were migrated `as is` from the original location.
I do override several of the wizard colors for dark theme so this won't prevent users from overwriting them in a custom theme.